spec/enumerable_spec.rb

```diff
@@ -0,0 +1,237 @@
+# frozen_string_literal: true
+
+require_relative '../enumerable_method'
+
+RSpec.describe Enumerable do
+  describe '#my_all?' do
+    it 'return true when all the numbers are greater than 6' do
+      expect([10, 20, 20, 40, 50].my_all? { |x| x > 6 }).to be_truthy
+    end
+
+    it 'return false when not all the numbers are greater than 6' do
+      expect([10, 20, 20, 40, 4].my_all? { |x| x > 6 }).to be_falsy
+    end
+
+    it 'return true if its equal as all? method' do
+      expect([10, 20, 20, 40, 50].my_all? { |x| x > 6 }).to eql([10, 20, 20, 40, 50].all? { |x| x > 6 })
```

Comment:
Note all of these methods are queries and the best way to test them is state explicitly what your expectation is.
```expect(test_array_one.my_all? { |x| x > 6 }).to eql(true)```.
Make sure you apply this any other place where this occurred